components/engine_rocks/src/db.rs

```diff
@@ -0,0 +1,218 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::fs::{self, File};
+use std::ops::Deref;
+use std::path::Path;
+use std::sync::Arc;
+
+use engine_traits::{
+    Error, IterOptions, Iterable, KvEngine, Mutable, Peekable, ReadOptions, Result, WriteOptions,
+};
+use rocksdb::{
+    set_external_sst_file_global_seq_no, DBIterator, IngestExternalFileOptions, Writable, DB,
+};
+use tikv_util::file::calc_crc32;
+
+use crate::options::{RocksReadOptions, RocksWriteOptions};
+use crate::util::{delete_all_in_range_cf, get_cf_handle};
+use crate::{Iterator, Snapshot, WriteBatch};
+
+#[derive(Clone)]
+#[repr(transparent)]
+pub struct Rocks(Arc<DB>);
+
+impl Rocks {
```

Comment:
why not `RocksDB`

Reply:
IMO, `Rocks` is shorter and could make sense too.😯